File: Resueltos_por_competidor/Santiago_Ortiz/BearAndBigBrother.java
import java.util.Scanner;
 
/*
 * Autor: Darkovski7   
 * Problema: Bear and Big Brother
 * Juez Online: Codeforces
 * Veredicto: Accepted 
 * Url: https://codeforces.com/problemset/problem/791/A
 */

public class BearAndBigBrother {
 
	public static void main(String[] args) {
 
		Scanner sc = new Scanner(System.in);
		
		int a;
		int b;
		int contador = 0;
		
		a = sc.nextInt();
		b = sc.nextInt();
		
		for (int i = 0; a <= b; i++) {
			contador = i;
			b = b*2;
			a = a*3;
		}
		System.out.println((contador+1));
	}
 
}

File: Resueltos_por_competidor/Martin_Peña/BearAndBigBrother.cpp
/*
* Autor: pgms1103-afk
* Problema: A. Bear and Big Brother
* Juez Online: Codeforces
* Veredicto: Accepted
* URL: https://codeforces.com/problemset/problem/791/A
*/
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int a;
    int b;
    cin >> a >> b;

    int contador = 0;
    while(a<=b){
        a = a*3;
        b = b*2;
        contador += 1;
    }
    cout << contador << endl;
    return 0;
}
